BaseLib/LogogCustomCout.h

```diff
@@ -25,21 +27,25 @@ namespace BaseLib
 class LogogCustomCout : public logog::Target
 {
 public:
-#ifdef USE_MPI
+#if defined(USE_MPI) || defined(USE_PETSC)
 	/**
 	 * Constructor when MPI is involved
 	 *
 	 * @param all_rank_output_level  Minimum level to output messages from all MPI processes
 	 * @param mpi_comm               MPI communicator
 	 */
+#if defined(USE_PETSC)
+	LogogCustomCout(LOGOG_LEVEL_TYPE all_rank_output_level = LOGOG_LEVEL_INFO, MPI_Comm mpi_comm = PETSC_COMM_WORLD)
```

Comment:
Debugging output is also available. I just committed some changes in the suppression flags. With the changed format, logog makes the same output as the in the master in serial computations, while in parallel computation it makes an output looking like
```
rank 0| debug: Parse process variables:
rank 0| debug: Constructing process variable pressure
rank 0| debug: Using value 0
rank 0| debug: GEOObjects::getGeoObject(): Could not find POINT "left"
```